figure1.R

```diff
@@ -0,0 +1,241 @@
+library(backports)
+
+sirs_gen <- odin.dust::odin_dust("sirs.R")
+sirs_times <- rep(NA_real_, 16)
```

Comment:
For resubmission I'd like to rework this to use bench::press + bench::mark which does a nice job of automating most of what is here.